File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/LotteryDrawVO.kt
package com.sdkwork.app

data class LotteryDrawVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val isWin: Boolean? = null,
    val prizeId: String? = null,
    val prizeName: String? = null,
    val prizeImage: String? = null,
    val prizeType: String? = null,
    val isGrandPrize: Boolean? = null,
    val winTime: String? = null,
    val recordId: String? = null,
    val remainingFreeCount: Int? = null,
    val pointsCost: Int? = null,
    val message: String? = null
)

File: sdkwork-app-sdk-typescript/src/types/lottery-draw-vo.ts
/** 抽奖结果 */
export interface LotteryDrawVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 是否中奖 */
  isWin?: boolean;
  /** 奖品ID */
  prizeId?: string;
  /** 奖品名称 */
  prizeName?: string;
  /** 奖品图片 */
  prizeImage?: string;
  /** 奖品类型 */
  prizeType?: string;
  /** 是否大奖 */
  isGrandPrize?: boolean;
  /** 中奖时间 */
  winTime?: string;
  /** 中奖记录ID */
  recordId?: string;
  /** 剩余免费次数 */
  remainingFreeCount?: number;
  /** 消耗积分 */
  pointsCost?: number;
  /** 提示消息 */
  message?: string;
}
